packages/app-project/src/screens/SubjectTalkPage/components/SubjectTalkViewer/SubjectTalkViewer.js

```diff
@@ -0,0 +1,91 @@
+import { Media } from '@zooniverse/react-components'
+import { Box } from 'grommet'
+import { Favorite, Bookmark, ShareOption, SubtractCircle } from 'grommet-icons'
+import dynamic from 'next/dynamic'
+import { useTranslation } from 'next-i18next'
+import { shape, string } from 'prop-types'
+import { useState } from 'react'
+
+// Dynamically import FlipbookControls with SSR disabled
```

Comment:
This is based on [the ClassifierWrapper import in the ClassifyPage](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/src/screens/ClassifyPage/ClassifyPage.js#L17-L19). Attempting to import FlipbookControls without this causes an error related to styled-components and grommet in lib-react-components' Media component, I think per the FlipbookControls use of the Thumbnail component. I'm not sure if there's a different change that can be made to avoid use of `dynamic` here and import in the typical fashion, please let me know if there is.